templates/default/eslintrc.json

```diff
@@ -19,6 +19,9 @@
   },
   "plugins": ["react"],
   "rules": {
-    "react/prop-types": ["error"]
+    "react/prop-types": ["error"],
```

Comment:
Let's review specifics please and see if these rule changes are actually necessary.